sdk/boms/azure-spring-cloud-dependencies/pom.xml

```diff
@@ -0,0 +1,278 @@
+
+  <groupId>com.azure.spring</groupId>
+  <artifactId>azure-spring-cloud-dependencies</artifactId>
+  <version>2.3.0</version>
+  <packaging>pom</packaging>
+
+  <name>Azure Spring Cloud dependencies BOM</name>
+  <description>BOM for Azure Spring Cloud dependencies</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <licenses>
+    <license>
+      <name>The MIT License (MIT)</name>
+      <url>http://opensource.org/licenses/MIT</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <id>microsoft</id>
+      <name>Microsoft Corporation</name>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git://github.com/azure/azure-sdk-for-java</connection>
+    <developerConnection>scm:git:git://github.com/azure/azure-sdk-for-java</developerConnection>
+    <url>https://github.com/azure/azure-sdk-for-java</url>
+  </scm>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/azure/azure-sdk-for-java/issues</url>
+  </issueManagement>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <packageOutputDirectory>${project.build.directory}</packageOutputDirectory>
+    <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
+    <spring.boot.version>2.4.3</spring.boot.version>
+    <spring.cloud.version>2020.0.1</spring.cloud.version>
+    <azure.spring.cloud.version>2.3.0</azure.spring.cloud.version>
+    <azure.cloud.appconfiguration.config>1.3.0</azure.cloud.appconfiguration.config>
+    <azure-resourcemanager.version>2.3.0</azure-resourcemanager.version>
+    <azure.eventhubs.version>5.6.0</azure.eventhubs.version>
+    <azure.eventhubs.checkpoint.store.version>1.5.1</azure.eventhubs.checkpoint.store.version>
+    <azure.storage.blob.version>12.10.0</azure.storage.blob.version>
+    <azure.storage.file.version>12.8.0</azure.storage.file.version>
+    <azure.storage.queue.version>12.8.0</azure.storage.queue.version>
+    <azure.servicebus.version>3.6.1</azure.servicebus.version>
+    <azure-core.version>1.14.1</azure-core.version>
+    <azure-identity-spring.version>1.3.0</azure-identity-spring.version>
+    <azure-data-appconfiguration.version>1.1.10</azure-data-appconfiguration.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>${spring.boot.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+
+      <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-dependencies</artifactId>
+        <version>${spring.cloud.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
Should we include this?

Reply:
It might be better to separate, let me remove it.